Dockerfile.template

```diff
@@ -50,9 +50,17 @@ RUN set -eux; \
 	} >> /usr/local/etc/gemrc
 
 ENV LANG C.UTF-8
+{{ if .version | . == "3.0.6" or . == "3.1.4" or . == "3.2.2" then ( -}}
```

Comment:
This was intentionally conservative to avoid churn in all the `Dockerfile`s for what amounts to entirely superficial changes.